src/backend/Level0/CHIPBackendLevel0.cc

```diff
@@ -590,6 +595,51 @@ CHIPQueueLevel0::CHIPQueueLevel0(CHIPDeviceLevel0 *ChipDev, unsigned int Flags)
 CHIPQueueLevel0::CHIPQueueLevel0(CHIPDeviceLevel0 *ChipDev)
     : CHIPQueueLevel0(ChipDev, 0, 0) {}
 
+void CHIPQueueLevel0::initializeEventPool(CHIPDeviceLevel0 *ChipDev) {
```

Comment:
Not used anywhere

Reply:
this is used in my original code; for some reason commit a521589e3214103 comments it out. The code was originally in `CHIPQueueLevel0` constructor, since i added a new constructor (with native handles), and both need to initialize the queue's event pool AFAICT, so i put it the shared code in `initializeEventPool`.